nextflow.config

```diff
@@ -251,15 +251,70 @@ dag {
 manifest {
     name            = 'nf-core/mcmicro'
     contributors    = [
-        // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
         [
-            name: 'TBD',
-            affiliation: '',
+            name: 'Jeremy Muhlich',
+            affiliation: 'Harvard University',
```

Comment:
@AlexBarbera Could you change yours and mine to `Harvard Medical School, Laboratory of Systems Pharmacology`?